### DISCUSSION TITLE
get the actual bitrate experienced by UE

### DISCUSSION THREAD

**Discussion by manoj1919** [aligungr/UERANSIM#355] [Q&A]
Is there a way to calculate and see the actual bitrate the UE is experiencing, with UERANSIM. Any work arounds are also welcome
Thanks
Manoj

**Comment by aligungr** [comment]
We don't have a builtin mechanism (currently), but since the UERANSIM exposes a TUN interface, you can use other common methods for monitoring the network adapter.

I think there are many tools to show throughput for a specific interface. Even ifconfig shows some of these stats as far as I know.

BR
